templates/foreman.service-overrides.erb

```diff
@@ -5,3 +5,11 @@ Environment=FOREMAN_HOME=<%= scope['foreman::app_root'] %>
 Environment=FOREMAN_PUMA_THREADS_MIN=<%= scope['foreman::config::min_puma_threads'] %>
 Environment=FOREMAN_PUMA_THREADS_MAX=<%= scope['foreman::foreman_service_puma_threads_max'] %>
 Environment=FOREMAN_PUMA_WORKERS=<%= scope['foreman::config::puma_workers'] %>
+# Puma worker_timeout, which specifies the time for each worker to check in
+# before it is assumed hanging and then restarted. This is not a request timeout.
+# If not specified here, the default value of 60 seconds is used.
```

Comment:
I don't like hardcoding defaults all over the place. Especially in comments. If they change, they're out of sync and misleading. Since it's normal that when it's not specified, the default is used so I'd leave this line out.
```suggestion
```

Reply:
I updated to remove all hardcoded values, and moved all comments into the conditional so they will only be visible in the template when the user has configured a value here.